modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/ProjectResource.java

```diff
@@ -184,6 +192,47 @@ public ProjectJson get( final @QueryParam("name") String projectNameValue )
         return doCreateJson( this.projectService.get( projectName ) );
     }
 
+    @GET
+    @Path("getTree")
+    public List<ProjectParentJson> getTree( final @QueryParam("name") String projectNameValue )
+    {
+        final List<Project> result = new ArrayList<>();
+        final Projects projects = adminContext().callWith( () -> this.projectService.list() );
```

Comment:
I think we need graph method in projectService. This one does not follow spec https://github.com/enonic/xp/issues/8413#issuecomment-737136515